{
  "contest": 427,
  "handle": "VulnViper",
  "risk": "3",
  "title": "Out-of-Bounds Error in getPositionsForCurator Function",
  "issueId": 215,
  "issueUrl": "https://github.com/code-423n4/2024-08-phi-findings/issues/215",
  "originalIssueId": 325,
  "originalIssueUrl": "https://github.com/code-423n4/2024-08-phi-validation/issues/325"
}